scss/bitstyles/atoms/button/button.stories.js

```diff
@@ -111,8 +111,21 @@ DefaultOutline.parameters = {
   ],
 };
 
+const baseTabArgs = {
+  colorVariant: ['tab'],
+  role: 'tab',
```

Comment:
`aria-selected` is not allowed on normal buttons, so a tab role must be added. And then the button needs to we wrapped in a tablist because tabs must be children of tablists only, they cannot stand alone.